### PR TITLE
Change TensorReference to EdgeReference for code clarity

### PR DESCRIPTION
**Pull request by Kh4L**
`TensorReference` is often mistaken for `Tensor` and `TensorList`.

Change the name for code readibility.

### CODE REVIEW

File: docs/examples/getting started.ipynb [hunk 1]
```diff
@@ -58,7 +58,7 @@
       " |  \n",
       " |  define_graph(self)\n",
       " |      # defined by the user to construct their graph of operations.\n",
-      " |      # this returns a list of output TensorReferences that we can\n",
+      " |      # this returns a list of output EdgeReference that we can\n",
```
Comment (ptrendx) [inline docs/examples/getting started.ipynb]: Could you regenerate this? I think this docstring looks different now.

Comment (Kh4L) [reply docs/examples/getting started.ipynb]: Done

**Comment by Kh4L** [comment]
Build 605063

**Comment by Kh4L** [comment]
Build 605159

**Pull request merged**